app/containers/Forms/ProductRowIdField.tsx

```diff
@@ -29,47 +29,51 @@ export const ProductRowIdFieldComponent: React.FunctionComponent<Props> = (
       ...props,
       schema: {
         ...props.schema,
-        enum: props.query.active.edges.map(({node}) => node.rowId),
-        enumNames: props.query.active.edges.map(({node}) => node.name)
+        enum: props.query.published.edges.map(({node}) => node.rowId),
+        enumNames: props.query.published.edges.map(({node}) => node.productName)
       },
       query: undefined
     }),
     [props]
   );
 
   /**
-   * Contains all products, split into arrays of active / inactive product IDs and names.
+   * Contains all products, split into arrays of published / unpublished product IDs and names.
    */
   const allProducts = useMemo(
     () => ({
-      activeProductIds: props.query.active.edges.map(({node}) => node.rowId),
-      inactiveProductIds: [
+      publishedProductIds: props.query.published.edges.map(
+        ({node}) => node.rowId
+      ),
+      unpublishedProductIds: [
         ...props.query.archived.edges.map(({node}) => node.rowId),
-        ...props.query.deprecated.edges.map(({node}) => node.rowId)
+        ...props.query.draft.edges.map(({node}) => node.rowId)
```

Comment:
draft products are not needed here, they will never be in an application (can't go back to "draft" status)